main.go

```diff
@@ -40,6 +41,48 @@ func main() {
 		res.Write(installScript.Bytes())
 		scriptLock.Unlock()
 	})
+	http.HandleFunc("/release/", func(res http.ResponseWriter, req *http.Request) {
+		<-scriptReady
+		reqArchive := ""
+		reqVersion := ""
+
+		fmt.Sscanf(req.URL.Path, "/release/%s", &reqArchive)
+		reqArchive = strings.Replace(reqArchive, "latest", strings.TrimRight(version.String(), "\n"), 1)
+
+		sequences := strings.Split(reqArchive, "_")
+		if len(sequences) != 4 {
+			writeShortResponse(res, http.StatusNotFound, "Not found")
+			return
+		}
+
+		reqVersion = sequences[1]
+		if reqArchive == "" || reqVersion == "" {
+			writeShortResponse(res, http.StatusNotFound, "Not found")
+			return
+		}
+
+		archiveUrl := fmt.Sprintf("https://github.com/Scalingo/cli/releases/download/%s/%s", reqVersion, reqArchive)
+
+		githubRes, err := http.Get(archiveUrl)
+		if err != nil {
+			log.Println("http get error:", err)
+			writeShortResponse(res, http.StatusBadRequest, "Bad request")
+			return
+		}
+		defer githubRes.Body.Close()
+
+		if githubRes.StatusCode != http.StatusOK {
+			writeShortResponse(res, http.StatusBadRequest, "Bad request")
+			return
+		}
+
```

Comment:
How am I supposed to know the length? I'm a bit confused since we are piping the binary